server/models/events.js

```diff
@@ -12,6 +12,17 @@ const getAllEvents = (limit = 10, page = 0) => {
   return db.pool.query(query);
 }
 
+const getSpecificEvent = (id) => {
+  const query = `
+    select *
+    from events
+    where id = ${id}
```

Comment:
fyi - Concatenating strings into queries could lead to injection attacks.

You could use a [parameterized query](https://node-postgres.com/features/queries#parameterized-query) instead.

i.e., replace `${id}` with `$1`, and then call `db.pool.query(query, [id])`.

Reply:
I was wondering why you weren't using string literals; I'll update for the next PR. Thanks.